Onboarding.md

```diff
@@ -0,0 +1,163 @@
+    local development environment.
+- [Go by Example](https://gobyexample.com/)
+    - Ideal for those who learn best by doing.
+- [The Go Playground](https://go.dev/play/)
+    - Sandbox for playing with Go code.
+- [Go Docs](https://golang.org/doc/)
+    - Landing page for many Go resources e.g. learning, concepts, etc.
+- [Effective Go](https://go.dev/doc/effective_go)
+    - Outlines community-accepted code style and best practices.
+- [Our Go Guide](https://github.com/EdgeCast/ec-sdk-go/blob/main/Go.md)
+    - Best practices and code style our team adheres to.
+
+## Developing for Terraform
+Once you have familiarized yourself with Go, you can learn how to develop a
+custom provider. You can find the Hashicorp Learn tutorial 
+[here](https://learn.hashicorp.com/collections/terraform/providers).
+
+## Local Environment Setup
+If you have a Pluralsight license, we recommend watching the Getting Started 
+With Go course listed above in the Learning Resources.
+
+Below is a breakdown of the setup:
+1. Install Go - [download](https://go.dev/dl/)
+2. Install Git - [download](https://git-scm.com/downloads)
+3. Install Visual Studio Code - [download](https://code.visualstudio.com/download)
+4. Set up Git access for your machine
+    - Use a Git client like [Sourcetree](https://www.sourcetreeapp.com/) or
+    [GitHub Desktop](https://desktop.github.com/)
+    - [Create an SSH Key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh) 
+    for your GitHub account and add it to your local machine.
+5. Install Visual Studio Code Extension
+    - In VS Code, open the Extensions menu and search for "Go". The developer 
+    should be "Go Team at Google".
+6. Install the HashiCorp Terraform Extension.
+    - In VS Code, open the Extensions menu and search for "Go". The developer 
+    should be "Hashicorp".
+7. Install any suggested Go and Terraform tools as prompted by Visual Studio
+8. Clone the [terraform-provider-edgecast](https://github.com/EdgeCast/terraform-provider-edgecast) 
+repository.
+8. Refer to the [README.md](README.md), [Contributing.md](Contributing.md), and 
+[Architecture.md](Architecture.md) files.
+
+## Development Workflow
+Create a branch off of main and begin coding!
+
+### Running Your Local Code
+Refer to the "Testing Your Local Code" section of the [readme](README.md).
+
+### Debugging
+Debugging Terraform code is a bit more complicated than just clicking Run and 
+Debug in VS Code. You must start the debugger and have it listen before running
+terraform.
+
+1. Create a launch.json profile to run `go` with the `debug` flag
+2. Start a `delve` session
+3. Set the `TF_REATTACH_PROVIDERS` environment variable using the output of the
+`delve` command.
+4. Run `terraform` as usual.
+
+Please carefully read through 
+[Hashicorp's article on debugging](https://www.terraform.io/plugin/sdkv2/debugging)
```

Comment:
I think this might be a better link since it covers more on debugging and also links to the sdkv2 specific page. Or perhaps both if you want a direct link to the above. 
https://www.terraform.io/plugin/debugging

Reply:
Updated